### PR TITLE
Initial

### PR DESCRIPTION
**Pull request by sfawcett123**
## Improve documentation
Moving documents from [Confluence](https://dfedigital.atlassian.net/wiki/spaces/SE/overview) and ensuring that they are up to date, as well as not duplicating information provided by [Technical Guidance](https://technical-guidance.education.gov.uk/)

Also providing a URL via Github_pages so they are easier to navigate https://dfe-digital.github.io/schools-experience/

## Review comments
1. I will not be impacting the application
2. They style I chose is just because I like it, I have no intention on working on the style, if you want it changed please raise a separate PR and involve a Jekyll/CSS guru
3. I do not have knowledge of the whole system, so further changes will be made by people who know their stuff

### CODE REVIEW

File: docs/documents/DevOps/workflows.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+title: Workflows
+nav_order: 20
+parent: DevOps
+layout: page
+---
+
+# Workflows
+
+## Overview
+Workflows are orchestraited via [github actions](https://docs.github.com/en/actions) which build, test and deliver our code to the [GOV.UK PaaS](https://www.cloud.service.gov.uk/).
```
Comment (saliceti) [inline docs/documents/DevOps/workflows.md:11]: Typo: orchestrated

File: docs/documents/DevOps/workflows.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+Guidance about the use of Github Actions can be found on the [DfE Technical Guidance](https://technical-guidance.education.gov.uk/infrastructure/automation/github-actions/#installation) page.
```
Comment (saliceti) [inline docs/documents/DevOps/workflows.md:13]: Link to top of the page: https://technical-guidance.education.gov.uk/infrastructure/automation/github-actions/

File: docs/documents/DevOps/workflows.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+3. **Security Tests** - Brakeman and Snyk security tests are executed
+4. **Cucumber Tests** - Various application specific [Cucumber](https://cucumber.io/) tests are executed.
+5. **Sonar Cloud** - The [Sonarqube](https://www.sonarqube.org/) scanner is executed and the results passed to the [Sonarcloud](https://sonarcloud.io/) quality gateway
+6. **Deployments**
+	7. **Review** - When a PR is created a review is required, and this deployment brings up the specific version of the application so it can be checked out by the reviewer. 
+	7.	**Development** - When the review is accepted and merged the application is first delivered to the Development area. This is to ensure any code that impacts the delivery workflow does not bring down the system.
+	8. **Quality Assurance** - Also known as the staging area, this is a near live test, ensuring all the components hang together, so there is a high degree of confidence the Production deployment will work.
+	9. **Production** - The final deployment to the live system   
+
+### [Check Service Principle](https://github.com/DFE-Digital/schools-experience/blob/master/.github/workflows/check_sp.yml)
```
Comment (saliceti) [inline docs/documents/DevOps/workflows.md:34]: Typo: Principal

File: docs/index.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+#### What is School Experience?
+
+When a potential teacher spends some time in a school, observing and experiencing teaching / school life
+
+
+#### How will the School Experience Service help? 
+
+The School Experience Service will help:
+
+more potential teachers easily find an appropriate school experience
```
Comment (saliceti) [inline docs/index.md:32]: Missing list formatting (try preview)

File: docs/documents/DevOps/workflows.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+Delivery to the [GOV.UK PaaS](https://www.cloud.service.gov.uk/) is via [Terraform](https://technical-guidance.education.gov.uk/infrastructure/hosting/govuk-paas/tooling/#terraform) which delivers [containers](https://technical-guidance.education.gov.uk/infrastructure/hosting/govuk-paas/tooling/#docker) built by the workflow and stored in [GitHub Container Registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry)(GHCR)
+
+## Workflows
+
+### [Build and Deploy](https://github.com/DFE-Digital/schools-experience/blob/master/.github/workflows/build.yml)
+
+This is the heart of the deployment system, it has a number of phases
+
+1.	**Build** - During this phase the docker image built and pushed to the GHCR
```
Comment (leoapost) [inline docs/documents/DevOps/workflows.md:23]: The indentation is a little off

File: docs/documents/DevOps/workflows.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+1. **Build** - During this phase the docker image built and pushed to the GHCR
+2. **Unit Tests** - Lint, Rubocop and Spec tests are executed
+3. **Security Tests** - Brakeman and Snyk security tests are executed
+4. **Cucumber Tests** - Various application specific [Cucumber](https://cucumber.io/) tests are executed.
+5. **Sonar Cloud** - The [Sonarqube](https://www.sonarqube.org/) scanner is executed and the results passed to the [Sonarcloud](https://sonarcloud.io/) quality gateway
+6. **Deployments**
+	1. **Review** - When a PR is created a review is required, and this deployment brings up the specific version of the application so it can be checked out by the reviewer. 
+	7. **Development** - When the review is accepted and merged the application is first delivered to the Development area. This is to ensure any code that impacts the delivery workflow does not bring down the system.
+	8. **Quality Assurance** - Also known as the staging area, this is a near live test, ensuring all the components hang together, so there is a high degree of confidence the Production deployment will work.
+	9. **Production** - The final deployment to the live system
```
Comment (leoapost) [inline docs/documents/DevOps/workflows.md:32]: Should these sub bullet points be 1,2,3,4?

**Pull request merged**